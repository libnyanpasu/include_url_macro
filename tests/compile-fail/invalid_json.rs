use include_url_macro::include_json_url;

fn main() {
    // This should fail because README.md is not valid JSON
    let _json =
        include_json_url!("https://raw.githubusercontent.com/rust-lang/rust/master/COPYRIGHT");
}
