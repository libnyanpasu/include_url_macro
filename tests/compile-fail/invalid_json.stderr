error: Invalid JSON content from URL: expected value at line 1 column 1
 --> tests/compile-fail/invalid_json.rs:6:9
  |
6 |         include_json_url!("https://raw.githubusercontent.com/rust-lang/rust/master/COPYRIGHT");
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `include_json_url` (in Nightly builds, run with -Z macro-backtrace for more info)
